providers/pipeliner.go

```diff
@@ -5,9 +5,10 @@ const Pipeliner Provider = "pipeliner"
 func init() {
 	// Pipeliner API Key authentication
 	SetInfo(Pipeliner, ProviderInfo{
+		// TODO [ExplicitWorkspaceRequired: true]
 		DisplayName: "Pipeliner",
 		AuthType:    Basic,
-		BaseURL:     "https://eu-central.api.pipelinersales.com",
+		BaseURL:     "https://eu-central.api.pipelinersales.com/api/v100/rest/spaces/{{.workspace}}/entities",
```

Comment:
Sorry if there was already a discussion that you had with Rajat (he's OOO for 2 weeks). We generally avoid adding version numbers to the base URL, so that builders who are using proxy action can use any version they'd like and are not constrained to the version we use. 

So can you revert the BaseURL to be "https://eu-central.api.pipelinersales.com", and then within the Read and Write methods you can add `/api/v100/rest/spaces/{{.workspace}}/entities`